src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -1299,7 +1299,27 @@ export class StudyViewPageStore {
             !this._clinicalDataBinFilterSet.get(uniqueKey)!.disableLogScale;
     }
 
-    @action addCharts(visibleChartIds:string[]) {
+    @autobind
+    @action
+    public updateCustomBins(uniqueKey: string, bins: number[]) {
+        let newFilter = _.clone(this._clinicalDataBinFilterSet.get(uniqueKey))!;
```

Comment:
Here is the observable map we discussed.

Reply:
@zhx828 we should examine why this causes entire studyview to reload.  i think this the the only action which takes place, right?  i agree with @jjgao that this should really not result in a reload.   i would be interested in getting to the bottom of it with you

Reply:
This issue has been resolved.